client/src/EstimatesComparison/EstimatesComparison.yaml

```diff
@@ -17,22 +17,26 @@ general_info:
+    Avant de dépenser des fonds publics, le gouvernement doit obtenir l'autorisation du Parlement, soit au moyen de lois adoptées auparavant qui accordent une autorisation permanente, soit au moyen d'un projet de loi de crédits annuel déposé et adopté au Parlement. Comme pour les autres projets de loi, les projets de loi de crédits deviennent des lois une fois qu'ils ont été approuvés par la Chambre des communes et le Sénat et qu'ils ont reçu la sanction royale.
 
-    Pour appuyer l’examen des projets de loi par le Parlement, le président du Conseil du Trésor dépose au Parlement un document du budget des dépenses (budget principal ou budget supplémentaire) qui fournit des renseignements et des détails sur les autorisations de dépenses demandées. Bien que le budget principal des dépenses fournisse un aperçu des besoins en matière de dépenses pour l’exercice à venir, le budget supplémentaire des dépenses présente de l’information sur les besoins supplémentaires en matière des dépenses qui n’étaient pas suffisamment étoffées au moment du dépôt du budget principal des dépenses ou encore qui ont été peaufinées par la suite pour prendre en compte les changements apportés à des programmes ou à des services particuliers.
+    Pour appuyer l'examen des projets de loi par le Parlement, le président du Conseil du Trésor dépose au Parlement un document du budget des dépenses (budget principal ou budget supplémentaire) qui fournit des renseignements et des détails sur les autorisations de dépenses demandées. Bien que le budget principal des dépenses fournisse un aperçu des besoins en matière de dépenses pour l'exercice à venir, le budget supplémentaire des dépenses présente de l'information sur les besoins supplémentaires en matière des dépenses qui n'étaient pas suffisamment étoffées au moment du dépôt du budget principal des dépenses ou encore qui ont été peaufinées par la suite pour prendre en compte les changements apportés à des programmes ou à des services particuliers.
+    
+    Dans des circonstances exceptionnelles, lorsque le Parlement n'est pas en session, des mandats spéciaux peuvent être émis pour autoriser des dépenses urgentes. Ces mandats sont de nature temporaire et doivent être remplacés par une loi de crédits une fois que le Parlement se réunit à nouveau.
 
 
 
 current_doc_this_year:
   transform: [handlebars]
   en: |
-   {{#if current_doc_is_mains}}Main Estimates{{else}}Supplementary Estimates {{current_sups_letter}}{{/if}} <span class="nowrap">{{est_in_year}}</span>
+   {{#if current_doc_is_mains}}Main Estimates{{else if current_doc_is_sw}}Canada Gazette, Part I, Volume 159, Number 15, April 12, 2025. Special Warrant, Order in Council P.C. 2025-462 of April 1, 2025{{else}}Supplementary Estimates {{current_sups_letter}}{{/if}} <span class="nowrap">{{est_in_year}}</span>
```

Comment:
Should this one also be switched to the short version?: 
![Screenshot 2025-04-23 at 4 01 51 PM](https://github.com/user-attachments/assets/12195746-0f75-4cd8-a60c-776edcf23231)

Reply:
Also the group "By Item Type" button in the estimate explorer is redirecting to an error page. I'm assuming this is happening because the stat items were removed

Reply:
Oops, thought I had switched, good catch